src/Block/Adminhtml/Product/Edit/AddToProjectButton.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyTranslate\Connector\Block\Adminhtml\Product\Edit;
+
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+
+class AddToProjectButton implements ButtonProviderInterface
```

Comment:
I'd say this should be `CreateProjectButton` for now as we always create a new project.